generator/src/main/java/com/scottlogic/deg/generator/Guice/BaseModule.java

```diff
@@ -50,7 +50,7 @@ protected void configure() {
 
         // Bind providers - used to retrieve implementations based on user input
         bind(DecisionTreeOptimiser.class).toProvider(DecisionTreeOptimiserProvider.class);
-        bind(Profile.class).toProvider(ProfileProvider.class);
```

Comment:
This binding isn't used. Profile isn't injected into any classes anymore.

The binding should be 

bind(ProfileProvider.class).in(Singleton.class);

or, more accurately 

bind(ProfileFactory.class)in(Singleton.Class);